uvicorn/workers.py

```diff
@@ -70,10 +71,6 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
         self.config = Config(**config_kwargs)
 
-    def init_process(self) -> None:
```

Comment:
Not sure with @graingert removed this from here, please make sure its ok :)

Reply:
the setup_event_loop method is removed, so we don't need to call it in init_process. We also don't need to have a method just to call super - that's the default